src/screens/NewFuse.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
```

Comment:
?

Reply:
had some variables that werent used yet (waiting for backend)